warehouse/malware/checks/utils.py

```diff
@@ -78,3 +78,7 @@ def extract_file_content(archive_stream, file_path):
                 return None
         except tarfile.TarError:
             return None
+
+
+class FatalCheckException(Exception):
```

Comment:
could this be moved to `warehouse/malware/checks/errors.py` or `warehouse/malware/errors.py`?